packages/postgres-database/src/simcore_postgres_database/migration/README.md

```diff
@@ -78,8 +80,8 @@ We create a revision script for the change by using the local db as follows:
 pip install -r packages/postgres-database/requirements/dev.txt # install sc-pg package
 docker-compose -f services/docker-compose.yml -f services/docker-compose-ops.yml up adminer # bring db and ui up
 docker ps # find the published port for the db
-sc-pg discover -u simcore -p simcore --port=32787 # discover the db
-sp-pg info # what revision are we at?
+sc-pg discover -u scu -p adminadmin --port=5432 # discover the db
```

Comment:
hem

Reply:
what does this mean?

Reply:
aren't you pushing some usernames-passwords here?

Reply:
Already pushed to upstream/master in .env-devel:
![image](https://user-images.githubusercontent.com/8209087/175051861-92ec0553-32be-4f8d-8307-309364d6f458.png)

Nothing is revealed that is not already there